src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -4062,6 +4063,19 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         SearchMissingCommand.raise(*this, args);
     }
 
+    winrt::fire_and_forget TermControl::_bubbleWindowSizeChanged(const IInspectable& /*sender*/, const Control::WindowSizeChangedEventArgs args)
+    {
+        co_await wil::resume_foreground(Dispatcher());
+
+        til::size cellSize{ args.Width(), args.Height() };

```

Comment:
I think it'd be easier to understand if this was named `cellCount` (we use `cellSize` for the size of a cell in pixels elsewhere).